src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,82 @@
+package pages;
+
+import org.openqa.selenium.*;
+
+public class HomePage extends BasePage {
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    By userAvatarIcon = By.cssSelector("img.avatar");
+    By createPlaylistBtn = By.xpath("//i[@data-testid='sidebar-create-playlist-btn']");
+    By createSimplePlaylistOption = By.xpath("//li[@data-testid='playlist-context-menu-create-simple']");
+    By playlistNameTextField = By.xpath("//section[@id='playlists']//input[@type='text']");
+    By parentSuccessMsg = By.cssSelector("div.success.show");
+    By searchField = By.cssSelector("input[type='search']");
+    By viewAllSongsBtn = By.cssSelector("button[data-test='view-all-songs-btn']");
+    By firstSongOnList = By.xpath("//section[@id='songResultsWrapper']//table[contains(@class, 'items')]//tr[1]");
+    By addToPlaylistBtn = By.cssSelector("button[class='btn-add-to']");
+    By playlistInDropdown = By.xpath("//section[@id='songResultsWrapper']//li[contains(@class,'playlist') and contains(text(),'New Playlist')]");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```